### DISCUSSION TITLE
The disappearing navigation bar is a bad UX decision

### DISCUSSION THREAD

**Discussion by scottjenson** [elk-zone/elk#1997] [General]
There are two problems:

1. I can't see if I have any notifications
2. I have to move the most to 'see' where I need to go

As a UX designer and this appears to be a visual design being more important than ux design. It *does* look cleaner but it comes at too high of a cost. I'd suggest turning it off but if you feel strongly it needs to be there, please allow it to be turned off in settings.

**Comment by patak-cat** [comment]
Maybe you have entered Zen mode? Press `z` to get out of it. There is a toggle button on the bottom right too to enable/disable it.

**Comment by scottjenson** [comment]
Thank you! I did! I had absolutely no idea what Zen mode was or how I got into it. After your comment, I looked around but the ONLY way to get back to normal was to type 'z' as you instructed and that worked. I wish there was more obvious way to get in/out of that mode. ESPECIALLY as I got into it by accident.

**Reply by scottjenson** [reply]
<img width="176" alt="image" src="https://user-images.githubusercontent.com/1055674/233507587-9427e65f-8edc-4582-87af-00e39ac1f771.png">
OK, finally found it. That is FAR too subtle. I'd suggest at the first least a visual toggle so the icon is inverted so very dark. That way when when I'm scanning around, it will draw my eye a bit more. As it is, I completely missed it.

**Reply by patak-cat** [reply]
Agreed, this is being tracked in this issue https://github.com/elk-zone/elk/issues/1883
